css/custom.css

```diff
@@ -0,0 +1,266 @@
+/* Custom CSS for Cloud Haskell Documentation */
+
+:root {
+  --primary-color: #5e81ac;
+  --primary-dark: #4c6a8f;
+  --primary-light: #88c0d0;
+  --secondary-color: #bf616a;
+  --dark-bg: #2e3440;
+  --light-bg: #eceff4;
+  --text-color: #2e3440;
+  --light-text: #eceff4;
+  --border-color: #d8dee9;
+  --code-bg: #3b4252;
+  --inline-code-bg: #e8f0f8;
+  --inline-code-color: #4c6a8f;
+  --inline-code-border: #d0dde9;
+}
+
+/* Typography */
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  color: var(--text-color);
+  line-height: 1.7;
+}
+
+.brand-text {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.5px;
+}
+
+/* Hero customization */
+.hero.is-primary {
+  background: linear-gradient(135deg, #5e81ac 0%, #88c0d0 100%);
+}
```

Comment:
I think the hero section should be a shade of purple, to match the Haskell logo. What do you think?